server/src/test/java/org/elasticsearch/search/rescore/QueryRescorerBuilderTests.java

```diff
@@ -200,11 +200,15 @@ public MappedFieldType fieldMapper(String name) {
         rescoreBuilder.setQueryWeight(randomFloat());
         rescoreBuilder.setRescoreQueryWeight(randomFloat());
         rescoreBuilder.setScoreMode(QueryRescoreMode.Max);
+        if (randomBoolean()) {
```

Comment:
I think we can always set the window size for this test.

Reply:
sure